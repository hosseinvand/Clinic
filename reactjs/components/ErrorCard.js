import React, {Component, PropTypes} from 'react'
import {Router, Route, Link, browserHistory, IndexRoute} from 'react-router'

class ErrorCard extends Component {
    constructor(props) {
        super(props)
    }

    render() {
        const {
            error
        } = this.props

        return (
            <div className="col-md-8 col-md-offset-2">
                <div className="alert alert-danger text-center">
                    <strong>{error}</strong>
                </div>
            </div>
        )
    }
}

ErrorCard.propTypes = {
    error: PropTypes.string.isRequired,
}

export default ErrorCard
