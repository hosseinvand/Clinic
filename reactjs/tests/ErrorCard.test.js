import React from 'react'
import ErrorCard from '../components/ErrorCard'
import renderer from 'react-test-renderer'

test('renders correctly', () => {
  const tree = renderer.create(
    <ErrorCard error="خطا خطا خطا" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});