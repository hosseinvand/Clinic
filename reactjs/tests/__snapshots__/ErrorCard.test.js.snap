exports[`test renders correctly 1`] = `
<div
  className="col-md-8 col-md-offset-2">
  <div
    className="alert alert-danger text-center">
    <strong>
      خطا خطا خطا
    </strong>
  </div>
</div>
`;
